components/app.js

```diff
@@ -29,5 +34,36 @@ class App{
 
     start(){
       this.getGrades()
+      gradeForm.onSubmit(this.gradeForm);
+    }
+
+    createGrade(name, course, grade){
+      console.log(name + " " + course + " " + grade);
```

Comment:
Please remove `console.log`s after you've verified the functionality they're testing works as expected.